### ISSUE TITLE
Revisit default styles for markdown

### ISSUE THREAD

**Issue by wwwillchen** [google/mesop#305]
In particular:

```css
.markdown * {
  margin: 0;
  padding: 0;
  list-style: none;
}
```

The `margin: 0` makes the markdown hard to read.

**Comment by wwwillchen** [comment]
Rename the CSS classname  from "markdown" to something more specific like "error-box-markdown", otherwise it becomes confusing what this is for. 

Context: https://github.com/google/mesop/commit/28c43dadccff5ea18a10f0a3f736c96df2251daf

**Comment by itsayellow** [comment]
In general the css style currently used for mesop from my reading is more like a "reset css" to be used as a base to apply style on top of.  (https://necolas.github.io/normalize.css/)  The formatting of this in general seems odd, especially with respect to margins around various text styles, and the zero-margin around the page in general.
Would it be possible to include a default style that is designed to look better, at least as an option to the developer to use?

**Comment by wwwillchen** [comment]
@itsayellow agree the default styles aren't super easy to work with. Are there specific changes to the default styles that you'd like to see?

The zero-margin around the page is intended to give the app developer greater flexibility, but this requires the app developer to often do some boilerplate to create a container with reasonable margins on the side, e.g. https://github.com/google/mesop/blob/ad19d572ce5b5661bc62b1b3442c3833b93b0993/mesop/labs/chat.py#L70-L71

I think we could provide some examples in our [demo gallery ](https://google.github.io/mesop/demo/) on setting up this container, but I don't think we will automatically add in margins/padding by default

**Comment by itsayellow** [comment]
In my mind I would think that this default style could remain the default, but there would be an alternate style(s) somehow available in the API or built-in files that would allow it to look good in a web browser out of the box.

For me, a lot of the header styles have strange spacings around them--the headers have no space above and a huge amount of space below their lines.

Additionally the zero-margin around the edges means that it needs extra work to look good as its own page, and not as a frame inside a bigger web page.

Lastly, just some general "prettiness" amongst the various styles would be nice, but this is the last priority.

Basically I use this kind of library because I know how to write Python and want to avoid making my own CSS or web design in general as much as humanly possible. 😄
